staging/src/k8s.io/apiserver/pkg/util/flowcontrol/dropped_requests_tracker_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+)
+
```

Comment:
I see a benchmark but no functional test.

Reply:
The benchmark is poor too.

yeah - the idea was to first converge on something that we roughly agree on - once we believe it's a good idea I will add tests.